CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- `workflow-event-topic` SNS topic, and `WorkflowEventManager` class for
```

Comment:
maybe clarify here that it's prefixed with the deployment name, since the code formatting indicates this is the exact name.